### PR TITLE
Use vars for registry image container name

### PR DESCRIPTION
**Pull request by csibbitt**
Values are here: https://github.com/infrawatch/service-telemetry-operator/blob/627db785aa1c5746dad6a7274106511f38694b3b/build/stf-run-ci/defaults/main.yml#L37

Waiting on PR #429

### CODE REVIEW

File: build/stf-run-ci/tasks/create_catalog.yml
Comment (csibbitt) [inline build/stf-run-ci/tasks/create_catalog.yml:79]: Despite the name "default", they never get used to create a "final" variable and are used as-is everywhere else in the code.

Comment (elfiesmelfie) [reply build/stf-run-ci/tasks/create_catalog.yml:79]: Some WIP on this in: https://github.com/infrawatch/service-telemetry-operator/pull/429/commits/cf2923eb3ee25081777da231be1a28effe1acc46

**Review by leifmadsen — APPROVED** [review]
I'm approving this, but I'm not sure if it should land or we should wait for the zuul changes to be reviewed and landed. Approving to unblock if that makes sense.

File: build/stf-run-ci/tasks/create_catalog.yml
Comment (csibbitt) [reply build/stf-run-ci/tasks/create_catalog.yml:79]: I'm in favor of any of these solutions in order of preference.

1. Start breaking your PR into digestible pieces so we can merge the improvements to this code independently
   * This likely has other benefits, that PR is getting very long lived and far-ranging
2. Merge this and adjust your PR accordingly
3. Abandon this and just assume we won't care to change this value until after that PR merges

I won't bother to apply any additional pressure here since I'm not fixing a problem, just noticed this and thought to be proactive.

Comment (csibbitt) [reply build/stf-run-ci/tasks/create_catalog.yml:81]: Hi there. I'm back after 3 weeks and I can't find any other PR that contains this change or an equivalent/replacement/conflicting change.  The original link to https://github.com/infrawatch/service-telemetry-operator/commit/cf2923eb3ee25081777da231be1a28effe1acc46 shows this (due to force-push I assume?)

image

I'm going to remove the do-not-merge label as I think this change can safely merge.

Comment (leifmadsen) [reply build/stf-run-ci/tasks/create_catalog.yml:81]: You'll get the "this commit does not believe to any branch on this repository" generally because when we merge branches, it squashes all the commits into a single commit rather than performing a merge commit. I noticed that as well in the past.

In this case though, you might be correct, because that commit refers to a parent commit in https://github.com/infrawatch/service-telemetry-operator/pull/470 so yea... I don't know where it went either :)

I agree this can likely merge without issue.

**Comment by csibbitt** [comment]
recheck

**Comment by csibbitt** [comment]
Two Jenkins' and a Zuul have approved this.

@elfiesmelfie do you have any objection to merging this? If you still have changes that conflict, I think they will be trivial enough to resolve.

**Comment by elfiesmelfie** [comment]
> Two Jenkins' and a Zuul have approved this.
> 
> @elfiesmelfie do you have any objection to merging this? If you still have changes that conflict, I think they will be trivial enough to resolve.

I can see where I'll need to make updates, it'll conflict, but I won't be thrown off by it

**Review by elfiesmelfie — APPROVED** [review]

**Pull request merged**
